vector/src/main/java/im/vector/app/features/home/room/detail/composer/MessageComposerAction.kt

```diff
@@ -34,6 +34,8 @@ sealed class MessageComposerAction : VectorViewModelAction {
     data class SlashCommandConfirmed(val parsedCommand: ParsedCommand) : MessageComposerAction()
     data class InsertUserDisplayName(val userId: String) : MessageComposerAction()
 
+    object ToggleFullScreen : MessageComposerAction()
```

Comment:
`ToggleFullScreen` maybe be replaced to a `data class SetFullScreen(val fullscreen: Boolean)`, because I think in the usage there is a View to enter fullscreen and a view to exit fullscreen. Also when the message is sent, it's maybe better to use `SetFullScreen(false)` rather than `if(fullscreen) ToggleFullScreen`.

But I have not tested to implement the change, this is maybe not better, I let you choose.